woocommerce/woocommerce
# Migrating attributes section in product editor to slot fills

### All Submissions:






### Changes proposed in this Pull Request:



Transitioning attributes section in product editor to utilize our slot-fill components. 

Closes #36418   .




### How to test the changes in this Pull Request:



1. Checkout branch.
2. Enable product editor at WooCommerce -> Settings -> Advanced -> Features -> Try the new product editor
3. Go to product editor at Products -> Add new (MVP)
4. To general smoke test of attributes section, ensure it behaves exactly the same as previously.



### Other information:




### FOR PR REVIEWER ONLY:
